src/algebra/algebra/subalgebra/basic.lean

```diff
@@ -347,10 +347,10 @@ def comap (S : subalgebra R B) (f : A →ₐ[R] B) : subalgebra R A :=
   .. S.to_subsemiring.comap (f : A →+* B) }
 
 theorem map_le {S : subalgebra R A} {f : A →ₐ[R] B} {U : subalgebra R B} :
-  map S f ≤ U ↔ S ≤ comap U f :=
+  map f S ≤ U ↔ S ≤ comap U f :=
```

Comment:
It would probably make sense to swap the arguments to comap at the same time, so that they remain consistent with each other.